Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -1524,5 +1534,22 @@ public List<string> SelectedTexts
     /// </remarks>
     [Parameter] public int? TotalItems { get; set; }
 
+    /// <summary>
+    /// Occurs after text has changed.
+    /// </summary>
+    [Parameter] public EventCallback<string> TextChanged { get; set; }
+    /// <summary>
```

Comment:
I don't want to sound too harsh. But always leave one empty line between properties

![image](https://github.com/Megabit/Blazorise/assets/900302/69edc04d-9dd2-40bf-80a7-5a2f04958b02)

Reply:
I like it also, Sometimes I just don't notice.

Reply:
Do you maybe have CodeLens enabled in Visual Studio?

Reply:
Yes, all sorts of stuff enabled... that sometimes you just have too many visual jargon that you stop noticing formatting as easily hehe